src/utils/pairSearch.ts

```diff
@@ -79,9 +86,11 @@ export const searchPairKeys = (
   transformSlugExp: RegExp = pairSearchExp
 ) => {
   // Transform search into pair
-  const searchSlug = fromPairSearch(search, transformSlugExp);
+  const _searchSlug = fromPairSearch(search, transformSlugExp);
+  const searchSlug = specialStringToRoman(_searchSlug);
```

Comment:
I think it's better to include this replacement in the `fromPairSearch` & `createPairMaps`.